pkg/asset/installconfig/gcp/session.go

```diff
@@ -18,8 +19,22 @@ import (
 var (
 	authEnvs            = []string{"GOOGLE_CREDENTIALS", "GOOGLE_CLOUD_KEYFILE_JSON", "GCLOUD_KEYFILE_JSON"}
 	defaultAuthFilePath = filepath.Join(os.Getenv("HOME"), ".gcp", "osServiceAccount.json")
+	credLoaders         = []credLoader{}
+	onceLoggers         = map[credLoader]*sync.Once{}
 )
 
+func init() {
```

Comment:
I would like to refrain from using init functions.

Reply:
Replaced with a lazy init function.